src/controllers/UserController.js

```diff
@@ -16,10 +16,10 @@ class UserController {
         password,
       },
     });
-    return result.data.user;
+    return Promise.resolve(result.data.user);
```

Comment:
what do you think about removing this code? it may even be confusing afterwards 🤔

Reply:
Agree with that, the developer should know how to handle this code.